website/deploy/index.html

```diff
@@ -76,9 +76,9 @@ <h2>Next steps</h2>
       </div>
     </section>
   </section>
-
   <footer>
-    <p class="fineprint">* Note: This is <b>NOT</b> an official Google product. Customers will be billed for using cloud resources when creating a sandbox cluster.
+    <p class="fineprint">* Note: This is <b>NOT</b> an official Google product. Customers will be billed for using cloud resources when creating a sandbox cluster. 
+    <button onclick="userfeedback.api.startFeedback({ 'productId': '5235454', 'productVersion': '0.7.0' });">Submit Feedback</button>
```

Comment:
It looks like you hardcoded the version number a couple places here. make-release.sh is supposed to auto-update all the version numbers in our repo, and it looks like [these files are tracked](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/7663c868026f9b49b2701df466ed3b5a756b9e1c/make-release.sh#L42), but:
- it expects the current version to be v0.6.0
- it expects the format to start with "v"

You change this to v0.6.0, and then try running  `./make-release.sh --dry-run --no-push` to make sure it re-writes the version properly